### DISCUSSION TITLE
[BP#79] Combiner les fichiers CSS et JavaScript

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#220] [Bonnes pratiques] [answered]
Un cas particulier de la [BP#009](https://github.com/cnumr/best-practices/blob/main/chapters/BP_009_fr.md) ?

Implementation spécifique Apache pas forcément utile à mon avis.

Il y a plusieurs cas comme cela? Ne faudrait-il pas systématiser soit de découper, soit de ne garder qu'une BP générale ?

**Comment by ACTLEM** [comment]
Apache est utilisé comme exemple, à l'époque ce n'était pas déconnant. A voir si webpack est plus terrain, réaliste aujourd'hui.
J'avoue que la BP 9 m'a toujours questionné car finalement il fait référence à d'autres BP et relate juste une attente de résultat/métrique. Je ne vois pas l'intérêt véritable de la BP 9 en tant que tel.

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé un issue pour rajeunir cette BP avec par exemple `webpack`: https://github.com/cnumr/best-practices/issues/267

@devcrafting je te l'ai affecté,  n'hésite pas si tu veux que je m'en occupes.
